src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -127,19 +147,23 @@ let inline private aidFlattening
     let outlierCount = flatSeed |> mixSeed "outlier" |> randomUniform outlierInterval
     let topCount = flatSeed |> mixSeed "top" |> randomUniform topInterval
 
-    let outliersSum = sortedAidContributions |> Seq.take outlierCount |> Seq.sumBy snd
-
     let topGroupSum =
       sortedAidContributions
       |> Seq.skip outlierCount
       |> Seq.take topCount
       |> Seq.sumBy snd
 
     let topGroupAverage = (float topGroupSum) / (float topCount)
-    let outlierReplacement = topGroupAverage * (float outlierCount)
+
+    let droppedContributions =
```

Comment:
this is pretty nitpicky, but we're not dropping contributions but altering them - understanding the word "contribution" as a whole "thing" that a given AID contributes. I have no elegant alternative though... `flatteningByAID` or `aidFlattenings` to match other terms aroung this file?

Reply:
These are the altered contributions that are not included in the aggregated result. They are removed from the total, so calling them "dropped" seems logical to me.